src/StreamOpt.cpp

```diff
@@ -669,6 +669,9 @@ class StreamOpt : public IRMutator {
 
             // insert line buffers for input streams
             for (const string &kernel_name : dag.input_kernels) {
+                internal_assert(dag.kernels.count(kernel_name)) << kernel_name
```

Comment:
It sounds like it should be a `user_assert`.

Reply:
Got it. Didn't know this function exists. I will change to `user_assert`.